lib/ftp.c

```diff
@@ -2482,9 +2482,9 @@ static CURLcode ftp_state_get_resp(struct Curl_easy *data,
           bytes--;
         }
         /* if we have nothing but digits: */
-        if(bytes++) {
+        if(bytes) {
           /* get the number! */
-          (void)curlx_strtoofft(bytes, NULL, 0, &size);
+          (void)curlx_strtoofft(++bytes, NULL, 0, &size);
```

Comment:
Minor nitpick: wouldn't it be more readable to just do `bytes++;` on its own line before the  `curlx_strtoofft` call?

Reply:
Fine for me too. Personally, I find both equally readable. What do others think?

Reply:
As much as I like terse code, `++bytes;` on a separate line makes it easier not to miss. It's not a strong opinion, however.

Reply:
I'm equally fine with either.

Reply:
I put it on a separate line in the final commit that landed on master.